analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -80,11 +79,10 @@ class Bundler(
         ) = Bundler(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
-    override fun command(workingDir: File?) = if (Os.isWindows) "bundle.bat" else "bundle"
+    override fun command(workingDir: File?) = "ruby"
```

Comment:
This breaks the version check when using jruby with the following message:

```
Exception in thread "main" com.vdurmont.semver4j.SemverException: Invalid version (no major version): jruby 9.1.17.0 (2.3.3) 2018-04-20 d8b1ff9 O
	at com.vdurmont.semver4j.Semver.<init>(Semver.java:61)
	at com.vdurmont.semver4j.Requirement.isSatisfiedBy(Requirement.java:547)
	at org.ossreviewtoolkit.utils.CommandLineTool$DefaultImpls.checkVersion(CommandLineTool.kt:100)
	at org.ossreviewtoolkit.analyzer.managers.Bundler.checkVersion(Bundler.kt:65)
	at org.ossreviewtoolkit.utils.CommandLineTool$DefaultImpls.checkVersion$default(CommandLineTool.kt:96)
	at org.ossreviewtoolkit.analyzer.managers.Bundler.beforeResolution(Bundler.kt:89)
	at org.ossreviewtoolkit.analyzer.PackageManager.resolveDependencies(PackageManager.kt:229)
	at org.ossreviewtoolkit.analyzer.Analyzer$analyzeInParallel$1$1$1.invokeSuspend(Analyzer.kt:131)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

Reply:
`gem env` may be a good way to get Ruby's version. For jruby-9.1.17.0 above, the output of `gem env` includes a line with `- RUBY VERSION: 2.3.3 (2018-04-20 patchlevel 0) [java]`. This may be easier to parse than `ruby --version`.

I haven't tested this with other versions though, and my knowledge of the Ruby ecosystem is very limited.